src/saga/utils/pty_process.py

```diff
@@ -597,7 +601,7 @@ def read (self, size=0, timeout=0, _force=False) :
                         if  size:
                             readsize = size-len(ret)
 
-                        buf  = os.read (f, _CHUNKSIZE)
+                        buf  = os.read (f, _CHUNKSIZE).decode()
```

Comment:
Are we *really* sure that this is what we want?  I don't think `read` implied `decode` before, did it?